community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -169,6 +168,8 @@ public KernelTransactionImplementation( StatementOperationParts operations,
         this.pool = pool;
         this.clock = clock;
         this.tracer = tracer;
+        this.storageStatement = storeLayer.acquireStatement();
```

Comment:
Since that method no longer acquires a statement, but merely creates one it should be renamed to `newStatement()`. Calling `acquire()` acquires it. It's quite internal and KTI is basically the only user so that slight inconvenience of this change is minimal btw. I'll do this rename.